charts/oes/templates/oes-sapor-cacerts.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
what is the purpose of this one?

Reply:
Our Sapor Server is expecting the ca cert of the spinnaker URL (If SSL is enabled for the Spinnaker). By default it is not getting added to cacert of the Sapor. So we have taken out the existing sapor cacerts file and appended the ca cert of our demo.opsmx.com and giving it to the pod as a config map.